proposals/draft/19-layered-model-of-singer

```diff
@@ -0,0 +1,137 @@
+- These influence the way that code is written for their specific language
+
+### Layer 5: Tooling/Orchestration/UX/Infrastructure
+This is the bread and butter of organizations that adopt singer. As far as standards go, this is not generally something that will be voted upon by the Singer communicty as a whole. Instead, this is the state that all proprietary code falls in and the specific product offerings of practitioners fit. Nothing in this space is expected to be open source, but the option is there for things that may qualify for best practices.
+
+This layer can be considered an analog to the "Application Layer" of the OSI layered model of networking.
+
+Principles:
+- Specific to a particular vertical or market space
+- Specific to a single ops approach for deployment or other closer-to-the-metal conecerns
+- Not generalizable without either losing value in the source product
+
+-----------------------
+
+## III. Additional Information
+
+### Which layer(s) of the Singer ecosystem does this proposal directly touch?
+
+Select all that apply:
+
+- [ ] Singer Specification - required capabilities and behaviors
+- [ ] Singer Specification - optional capabilities and behaviors
+- [ ] Singer best practices and other guidance
+- [x] **Singer Working Group - practices and procedures**
+- [ ] Singer documentation (Other)
+
+### Are there any downsides to this change?
+
+There are certainly downsides to this change. Directing the conversation with a categorical model is very important to consider. There are potentially nuances that are not yet considered that must be accounted for in this proposal, but the author intends that those pieces can be amended to clarify as they come out of working together.
+
+### Is the change backwards compatible?
+
+There is nothing to be backwards comptible with. This is a first language proposal of this mechanism purely for the Working Group's consideration.
+
+### How are Singer developers affected by the change (if applicable)?
+
+Hopefully not at all unless there is a good reason to. A primary goal of this is to only require things that are truly worthwhile to implement for the general Singer world as a whole.
+
+### How are Singer users affected by the change? (if applicable)?
+
+Singer users should be able to utilize this language as well to evaluate things that claim to implement the Singer best practices.
+
+### Future Plans
+
+The future plans for this proposal are that the Working Group will continue to iterate upon it and come to truly beneficial categorizations that advance the world of data as a whole.
+
+### Excluded Alternatives
+
+A purely free approach to extension proposals has been explicitly excluded by this. An entirely bottom-up approach isn't compatible with this sort of structure.
+
+### Acknowledgements
+
+Thank you to Taylor Murhy and Aaron Steers from Meltano for engaging with the initial Issue associated with this idea and helping me develop it conceptually over the past months.
```

Comment:
```suggestion
Thank you to Taylor Murphy and Aaron Steers from Meltano for engaging with the [initial Issue](https://github.com/MeltanoLabs/Singer-Working-Group/issues/19) associated with this idea and helping me develop it conceptually over the past months.
```

Reply:
Thanks, I need to do a full typo pass on this thing 😂